model/bootstrap/node_info.go

```diff
@@ -90,6 +90,10 @@ type NodeInfo struct {
 	networkPrivKey crypto.PrivateKey
 	stakingPubKey  crypto.PublicKey
 	stakingPrivKey crypto.PrivateKey
+
+	// node machine account keys
+	// machinePubKey  crypto.PublicKey
+	machinePrivKey crypto.PrivateKey
```

Comment:
I don't think we will need this in the public representation. Probably safe to omit for now.